features/org.wso2.carbon.identity.captcha.server.feature/resources/conf/captcha-config.properties.j2

```diff
@@ -57,5 +57,8 @@ recaptcha.request.wrap.urls={{recaptcha.request_wrap_urls}}
 # recaptcha v3 score threshold
 recaptcha.threshold={{recaptcha.threshold}}
 
+# recaptcha enterprise score threshold to issue warn logs
```

Comment:
Can we have a default value for this like 0 having no value can trigger an exception right when parsing double?

Reply:
It won't parse double if the value is blank. It will take the default value. https://github.com/wso2-extensions/identity-governance/pull/768/files#diff-fe33aa72259ad318a3b1fdfcffcf1eed48c89830dc2f8e703b53fb18d377d784R668